modules/aggregation-layers/src/screen-grid-layer/screen-grid-layer.js

```diff
@@ -75,7 +75,7 @@ export default class ScreenGridLayer extends GridAggregationLayer {
     super.updateState({oldProps, props, changeFlags});
 
     const cellSizeChanged = props.cellSizePixels !== oldProps.cellSizePixels;
-    const dataChanged = this._isAggregationDirty({oldProps, props});
+    const dataChanged = this._isAggregationDirty({oldProps, props, changeFlags});
```

Comment:
Can we avoid exploding and reassembling the argument (`{oldProps, props, changeFlags}`) multiple times

Reply:
Updated.